src/Components/SearchResults.js

```diff
@@ -0,0 +1,46 @@
+import React, { useState, useEffect } from "react";
+import moment from "moment";
+import TableHead from "./TableHead";
+import TableRow from "./TableRow";
+import CustomerProfile from "./CustomerProfile";
+
+function SearchResults({ results, newCustomerInfo }) {
+  const [customerID, setCustomerID] = useState(0);
+  const [allCustomers, setAllCustomers] = useState(results);
+
+  useEffect(() => {
+    if (newCustomerInfo) {
+      newCustomerInfo.id = allCustomers.length + 1;
+      let tempvar = allCustomers.concat(newCustomerInfo);
+      setAllCustomers(tempvar);
+    }
+  }, [newCustomerInfo]);
+  console.log(newCustomerInfo);
+
+  return (
+    <div>
+      <table className="table">
+        <TableHead />
+        <tbody>
+          {allCustomers.map((item, index) => {
+            const nightsSpent = moment(item.checkOutDate).diff(
+              item.checkInDate,
+              "days"
+            );
+            return (
+              <TableRow
+                item={item}
+                key={index}
+                nightsSpent={nightsSpent}
+                setCustomerID={setCustomerID}
+              />
+            );
+          })}
+        </tbody>
+      </table>
+      <CustomerProfile id={customerID} />
```

Comment:
You could set `customerID` initially to null in state and then render the `CustomerProfile` component conditionally:
```
{customerID && <CustomerProfile id={customerID} />}
```
Without this it tries to make an API call with 0:

<img width="439" alt="Screenshot 2020-07-30 at 22 15 46" src="https://user-images.githubusercontent.com/38005641/88975561-4f059900-d2b2-11ea-94b7-73c0f9034f4a.png">

Reply:
done